dot/peerset/peerset_test.go

```diff
@@ -25,96 +29,87 @@ func TestPeerSetBanned(t *testing.T) {
 
 	// we ban a node by setting its reputation under the threshold.
 	rpc := newReputationChange(BannedThresholdValue-1, "")
+
 	// we need one for the message to be processed.
+	processor.EXPECT().Process(Message{Status: Drop, setID: 0x0, PeerID: "testPeer1"})
```

Comment:
There isn't any assertion after this point. Is this deliberate?

Reply:
Added more assertions!